stanovy.tex

```diff
@@ -71,34 +71,42 @@ \section{Činnost Spolku}
 \section{Členství}
 \begin{enumerate}[itemsep=0pt]
     \item Členství ve Spolku je dobrovolné.
-    \item Rozlišuje se dvojí členství:
+    \item Rozlišují se následující typy členství:
     \begin{enumerate}[itemsep=0pt,topsep=0pt]
         \item sympatizující členství,
-        \item aktivní členství.
+        \item aktivní členství,
+	\item emeritní členství.
     \end{enumerate}
-    \item Sympatizujícím i aktivním členem Spolku se může stát každá fyzická 
+    \item Za člena Spolku se považuje osoba s libovolným typem členství z výše uvedených.
+    \item Členem Spolku se může stát každá fyzická 
     osoba starší 18 let, bez rozdílu pohlaví, vyznání, politického a sociálního 
     zařazení, národnosti, rasy a státní příslušnosti. Členství se váže na osobu 
     člena, je nepřevoditelné na jinou osobu a nepřechází na jeho 
     právního nástupce. 
-    \item Fyzická osoba se stává sympatizujícím nebo aktivním členem ke dni 
+    \item Fyzická osoba se stává členem ke dni 
     rozhodnutí výboru Spolku o přijetí její žádosti o členství. Výbor Spolku, 
     může odmítnout přijetí žadatele o členství, pokud dojde k většinovému 
     názoru, že přijetím žadatele za člena by bylo nebo mohlo být ohroženo 
     řádné plnění poslání Spolku, případně že žadatel nejeví dostatečný zájem 
     o naplňování poslání Spolku.
     \item Ukončení členství:
     \begin{enumerate}[itemsep=0pt,topsep=0pt]
-        \item dobrovolné vystoupení člena – členství končí dnem doručení 
-        písemného oznámení o ukončení členství výboru,
+        \item dobrovolným vystoupení člena – členství končí dnem doručení 
+        oznámení o ukončení členství výboru,
         \item úmrtím člena,
         \item vyloučením člena – členství končí dnem doručení rozhodnutí 
         o vyloučení člena ze Spolku; výbor Spolku má právo vyloučit člena, 
         pokud svým jednáním porušuje zásady a cíle Spolku, zanedbává členské 
         povinnosti, ohrožuje řádně plnění poslání Spolku nebo se již déle než 
         rok nepodílí na naplňování poslání Spolku.
     \end{enumerate}
-    \item Právo sympatizujících i aktivních členů:
+    \item Změna typu členství:
+    \begin{enumerate}[itemsep=0pt,topsep=0pt]
+        \item žádostí člena - typ členství je změněn schválením výborem Spolku,
+	\item žádostí výboru Spolku - typ členství je změněn souhlasem dotčeného člena nebo marným uplynutím čtrnáctidenní lhůty po dni doručení žádosti,
+	\item rozhodnutím výboru Spolku v případě, že jde o člena s aktivním členstvím, který se již déle než rok nepodílí na naplňování poslání Spolku. 
+    \end{enumerate}
```

Comment:
Doteď nebylo vůbec řešeno, takže těch pár změn proběhlo ve vakuu. Tohle dává možnost měnit typy členství po schválení výborem -- řeší to problém obecně, není třeba tedy definovat které přechody jsou automatické a které musí výbor schovalovat + dává možnost výboru vyřadit nečinné aktivní členy.